Makefile

```diff
@@ -103,7 +103,7 @@ operator-sdk:
 	chmod 0755 operator-sdk
 
 # Build the container image
-container-build: unittest bundle
+container-build: kustomize unittest bundle
```

Comment:
So I see we forgot to call the `kustomize` makefile target. Please move it inside the `bundle` target instead of `container-build`.

Reply:
sure, done.